spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1129,11 +1141,13 @@ def process_completion(self, params):
         position, automatic = args
         try:
             completions = params['params']
+            stats = params['stats']
+            stats = {src: stats[src] for src in stats if stats[src] > 0}
+            include_fallback = len(stats) == 1 and 'fallback' in stats

```

Comment:
Same here, where `fallback` is now in lowercase. This makes the codee harder to reason about